app/client/styles/main.styl

```diff
@@ -399,18 +398,23 @@ section[name=signature] canvas
   border: 1px solid $color-black-soft
   border-radius: 3px
 
-.buttons a
-  -webkit-appearance: none
-  -moz-appearance: none
-  background-color: $color-compliment-2
-  color: $color-white-bright
-  border-color: $color-white-soft
+.buttons 
   display: block
-  font-size: 1rem
-  padding: $margin-small
-  margin-bottom: $margin-tiny
-  border-radius: 3px
+  a
+    background-color: $color-compliment-2
+    color: $color-white-bright
+    border-color: $color-white-soft
+    display: inline-block
+    font-size: 1rem
+    padding: $margin-small
+    margin-bottom: $margin-tiny
+    border-radius: 3px
+    cursor: pointer
   
+.upload-container 
+  input[type=file]:disabled ~ .buttons 
```

Comment:
I still dont really like this as it is a slightly weird selector [giving the buttons or sections a data attr would allow a more normal hierarchical selector] but its not that deep and i'm gonna merge